update.json

```diff
@@ -3,8 +3,8 @@
     "remote-settings-devtools@mozilla.com": {
       "updates": [
         {
-          "version": "1.9.0buildid20240422.103808",
-          "update_link": "https://github.com/mozilla-extensions/remote-settings-devtools/releases/download/1.9.0-build1/remote-settings-devtools.xpi"
+          "version": "1.10.0_TBD",
```

Comment:
Why `_TBD` in the version string here? We didn't do that in the past.

Reply:
As I wrote elsewhere, note for future readers: 

> The addon update mechanism will fetch this file. Its URL is specified here in the addon manifest: https://github.com/mozilla-extensions/remote-settings-devtools/blob/a0c22b367926e336d2198213cba7ded244def495/extension/manifest.json#L16
>
> The problem is that we can't know this version string in advance. It is set by Taskcluster when the final .xpi is signed. So, instead of setting it to something that looks final like I did in the past, I thought it would make it clearer that we need a PR [like this one](https://github.com/mozilla-extensions/remote-settings-devtools/pull/198) once the .xpi is going to be published